.github/copilot-instructions.md

```diff
@@ -0,0 +1,212 @@
+
+### Repository Statistics
+- **Size**: Medium-large multiplatform project with ~15 modules
+- **Language**: Kotlin (multiplatform)
+- **Build System**: Gradle with Kotlin DSL
+- **Target Platforms**: JVM, Android, iOS, JS (DOM), Wasm-JS (Canvas)
+- **Key Dependencies**: Compose Multiplatform, Kobweb Silk, Material Web Components
+- **Current Version**: 0.6.0-SNAPSHOT
+
+## Build Instructions and Environment Setup
+
+### Prerequisites
+- **JDK 17**: Required (as specified in GitHub Actions workflows)
+- **Network Access**: Required for initial builds to download Android Gradle Plugin and other dependencies
+- **Gradle 9.1.0**: Automatically handled by wrapper
+
+**IMPORTANT**: Initial builds require internet access for dependency resolution. Once dependencies are cached, subsequent builds can use `--offline` flag if needed.
+
+**IMPORTANT**: If the project uses snapshot dependencies of other `com.huanshankeji` libraries, especially in a branch other than `main` such as `dev`, refer to the setup instructions at <https://github.com/huanshankeji/.github/blob/main/dev-instructions.md#about-snapshot-dependencies-of-our-library-projects>.
+
+### Build Commands (In Order)
+
+#### 1. Basic Project Setup
+```bash
+./gradlew --version  # Verify Gradle 9.1.0 and JDK 17
+```
+
+#### 2. Publishing to Local Maven (Primary Development Command)
+```bash
+./gradlew publishToMavenLocal
+```
+**Important**: Always run `publishToMavenLocal` first when making changes, as mentioned in CONTRIBUTING.md. This publishes libraries to your local Maven repository so dependent projects can use your changes.
+
+#### 3. Running Tests and Checks (Essential Commands - validated and working)
+```bash
+./gradlew check
+```
+**Note**: The project has "limited number of tests" according to CONTRIBUTING.md, but this validates what exists.
+
+#### 4. Building Demo Applications
+```bash
+# Build the side-by-side demo (used for GitHub Pages)
+./gradlew :compose-multiplatform-html-unified-demo:sideBySideBrowserDistribution
+
+# Individual platform builds
+./gradlew jsBrowserDistribution    # JS DOM version
+./gradlew wasmJsBrowserDistribution  # Wasm-JS Canvas version
+```
+
+### Build Configuration Details
+
+#### Memory Requirements
+- **Gradle JVM**: 2GB memory allocated via `org.gradle.jvmargs=-Xmx2G` in gradle.properties (required for Wasm compilation)
+
+#### Common Build Issues and Workarounds
+
+1. **Network Connectivity**: Initial builds require internet access for Android Gradle Plugin (8.11.2) and other dependencies. If build fails with network errors, ensure internet connectivity.
+
+2. **Memory Issues**: If Wasm compilation fails with OOM, the memory is already optimized in gradle.properties.
+
+3. **Kotlin/JS Store**: The `kotlin-js-store` directory may be generated during JS builds - this should not be committed.
```

Comment:
`kotlin-js-store` should be commited when updated automatically by Gradle commands. Where did you get this that it "should not be committed"?